packages/builder-util/src/fs.ts

```diff
@@ -306,6 +306,25 @@ export function copyDir(src: string, destination: string, options: CopyDirOption
   }).then(() => BluebirdPromise.map(links, it => symlink(it.link, it.file, symlinkType), CONCURRENCY))
 }
 
+export async function dirSize(dirPath: string): Promise<number> {
+  const entries = await readdir(dirPath, { withFileTypes: true })
+
+  const entrySizes = entries.map(async entry => {
+    const entryPath = path.join(dirPath, entry.name)
+
+    if (entry.isDirectory()) return await dirSize(entryPath)
```

Comment:
Please run `pnpm prettier`. That should resolve this. If not, please wrap this in `{ }`
```
if (entry.isDirectory()) {
   return await dirSize(entryPath)
}
```

Please run `pnpm prettier` regardless though :)

Reply:
Done